package.json

```diff
@@ -11,7 +11,8 @@
     "dev": "tsc-watch --onSuccess \"node ./dist/index.js\"",
     "eslint": "eslint src tests",
     "test": "mocha -r ts-node/register tests/**/*.test.ts --slow 0",
-    "testtxhist": "mocha -r ts-node/register tests/txHistory.test.ts",
+    "test:file": "mocha -r ts-node/register ",
+    "testtxhist": "npm run test:file -- tests/txHistory.test.ts",
```

Comment:
Unrelated: add new script `test:file` to run a specific test file(s) which can be passed as a parameter using npm. e.g. `npm run test:file -- test/txHistory.test.ts`